### PR TITLE
[16.0][MIG] repair_reinvoice: Migration to 16.0

### PR DESCRIPTION
**Pull request by PauBForgeFlow**

### CODE REVIEW

File: repair_reinvoice/hooks.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                [
+                    ("move_type", "=", "out_refund"),
+                    ("reversed_entry_id", "=", invoice_id.id),
+                ]
+            )
+            invoiced_repair.invoice_ids += refunds
+
+
+def post_init_hook(cr, registry):
+    with api.Environment.manage():
```
Comment (BernatPForgeFlow) [inline repair_reinvoice/hooks.py]: Since Odoo 15.0, Environment.manage() is useless. Maybe better to delete the line.

**Review by BernatPForgeFlow — CHANGES_REQUESTED** [review]

File: repair_reinvoice/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Follow the Odoo standard repair module flow. When an invoice related to a MO is reversed,
```
Comment (OriolMForgeFlow) [inline repair_reinvoice/readme/USAGE.rst]: Follow the Odoo standard repair module flow. When an invoice related to a **RO** is reversed,

**Review by OriolMForgeFlow — COMMENTED** [review]
I have been testing it on Runboat, and it doesn't seem to be working properly yet. 
I've created the Repair Order, confirmed it, started and finished it. I've created an Invoice, created a Reverse Invoice (both invoices with confirmed payment), and the option to create a new Invoice doesn't show up again.

Screenshot from 2023-08-25 09-02-01

Screenshot from 2023-08-25 09-02-19

**Review by OriolMForgeFlow — APPROVED** [review]
Code & Functional Review, LGTM!

**Review by mariadforgeflow — APPROVED** [review]
LGTM!

**Review by LoisRForgeFlow — APPROVED** [review]
/ocabot merge nobump

File: repair_reinvoice/models/repair_order.py [hunk 3]
```diff
@@ -52,6 +52,7 @@ def _create_invoices(self, group=False):
         for repair_id in repair_dict:
             repair = self.env["repair.order"].browse(repair_id)
             repair.invoice_ids += repair.invoice_id
+        return
```
Comment (LoisRForgeFlow) [inline repair_reinvoice/models/repair_order.py]: I just aborted the merge, you need to return the repair_dict here.

**Comment by LoisRForgeFlow** [comment]
@PauBForgeFlow could you attend my last comment? Thanks!

**Comment by LoisRForgeFlow** [comment]
Thanks!

/ocabot merge nobump

**Pull request merged**